extensions/objects/templates/FilepathEditor.js

```diff
@@ -163,6 +188,46 @@ oppia.directive('filepathEditor', [
           $scope.clearScopeData();
         };
 
+        $scope.generateResampledImageFile = function() {
+          var dataURIToBlob = function(dataURI) {
+            // Convert base64/URLEncoded data component to raw binary data
+            // held in a string.
+            var byteString;
+            if (dataURI.split(',')[0].indexOf('base64') >= 0) {
+              byteString = atob(dataURI.split(',')[1]);
+            }
+            else {
+              byteString = unescape(dataURI.split(',')[1]);
```

Comment:
Looked around for alternative implementations. I believe the unescape part was unnecessary. I've tested with different types of images again and it works.